packages/frontend/.env.development

```diff
@@ -0,0 +1,3 @@
+# This file should be the same for all developers
```

Comment:
We shouldn't add env files to git. Instead make it a template file and add instructions in README for copying to .env file.
Take a look at #11 for example.

Reply:
https://create-react-app.dev/docs/adding-custom-environment-variables/

Sorry I can't direct link to the line but

> .env files should be checked into source control (with the exclusion of .env*.local).

Reply:
oh ok I guess that makes sense :+1: